jupyterlab/labhubapp.py

```diff
@@ -18,22 +18,10 @@ def _default_url(self):
             return "/lab"
 
         def init_webapp(self, *args, **kwargs):
+             warnings.warn(
+                "SingleUserLabApp is deprecated, use SingleUserNotebookApp and set " + "c.Spawner.default_url = '/lab' in jupyterhub_config.py", DeprecationWarning
+            )
             super().init_webapp(*args, **kwargs)
-            settings = self.web_app.settings
-            if 'page_config_data' not in settings:
-                settings['page_config_data'] = {}
-            settings['page_config_data']['hub_prefix'] = self.hub_prefix
-            settings['page_config_data']['hub_host'] = self.hub_host
-            settings['page_config_data']['hub_user'] = self.user
-            api_token = os.getenv('JUPYTERHUB_API_TOKEN')
-            if not api_token:
-                api_token = ''
-            if not self.token:
```

Comment:
Is this handling of the no `api_token` case no longer necessary?

Reply:
It wasn't used in the original.

Reply:
Oh wait, I'm not sure

Reply:
Ah, this token doesn't matter because we're using hub authentication.